queries.sql

```diff
@@ -19,6 +19,7 @@ SELECT * FROM animals WHERE name Like '%mon';
 --  Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 ;
 
+-- day 2 question
 -- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
 - BEGIN TRANSACTION;
 - UPDATE animals SET species = 'unspecified';
```

Comment:
- [ ] Please avoid too many comments in your code. As stated in other files, details of your codebase should be in your README and pull request and not in the codebase. Kindly adhere to this to build brain muscle in writing clean code 👍